qCC/mainwindow.h

```diff
@@ -59,592 +59,2410 @@ class ccTracePolylineTool;
 
 struct dbTreeSelectionInfo;
 
-namespace Ui {
+namespace Ui
+{
 	class MainWindow;
-} 
+}
 
 //! Main window
-class MainWindow : public QMainWindow, public ccMainAppInterface, public ccPickingListener 
+/**
+ * \class MainWindow
```

Comment:
@dgirardeau I've changed the syntax for the comments. I'll do the same with the files in qcc, I'm nearly done with all the header files in the Qcc/ folders
I'll change it to the backslash syntax

Reply:
Yes, I personally prefer /** over /*!, that's perfect